pepper-apis/src/main/java/org/broadinstitute/ddp/json/Profile.java

```diff
@@ -38,6 +39,8 @@
     private String lastName;
     @SerializedName(BIRTH_DATE)
     private String birthDate;
+    @SerializedName(SHOULD_SKIP_LANGUAGE_POPUP)
+    private Boolean shouldSkipLanguagePopup;
```

Comment:
For consistency, can you also change this to `skipLanguagePopup`? Afterwards, can you also update the OpenAPI docs? There should be a Profile object in `schemas.yml`.